<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportWizard</class>
 <widget class="QWizard" name="ExportWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/tulip/gui/icons/logo32x32.png</normaloff>:/tulip/gui/icons/logo32x32.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="options">
   <set>QWizard::ExtendedWatermarkPixmap|QWizard::HaveFinishButtonOnEarlyPages|QWizard::HaveHelpButton|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <property name="titleFormat">
   <enum>Qt::RichText</enum>
  </property>
  <property name="subTitleFormat">
   <enum>Qt::RichText</enum>
  </property>
  <widget class="QWizardPage" name="choosePluginPage">
   <property name="title">
    <string>&lt;img src=&quot;:/tulip/gui/icons/64/document-export.png&quot; width=32 height=32/&gt; Graph export Wizard</string>
   </property>
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;br/&gt;Export the currently selected graph in the &lt;b&gt;Graphs&lt;/b&gt; list using one of the export methods listed below.&lt;br/&gt;Adjust its parameters if needed. Once you're done, click &lt;b&gt;OK&lt;/b&gt; to export your graph.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="exportFileLabel">
        <property name="text">
         <string>Export file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ClearableLineEdit" name="pathEdit">
        <property name="toolTip">
         <string>Set the full path of the file where you want to save the graph or click on the browse button</string>
        </property>
        <property name="placeholderText">
         <string>Set the full path of the file where you want to save the graph or click on the browse button</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Choose the export file location</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/tulip/graphperspective/icons/16/document-open.png</normaloff>:/tulip/graphperspective/icons/16/document-open.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="algFrame">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
	<string notr="true">
#categoryFrame, #parametersFrame {
border: 1px solid #C9C9C9;
}

#categoryLabel, #parametersLabel {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
stop:1 rgba(0,0,0,0),
stop:0 #F0F0F0);
border: 0px;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
	<property name="margin">
          <number>0</number>
	</property>
       <item>
        <widget class="QFrame" name="categoryFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>30</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="leftLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="categoryLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;b&gt;Category&lt;/b&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="exportModules">
            <property name="styleSheet">
             <string notr="true">QTreeView:enabled { color: black; }

 QTreeView::branch {
         border-image: url(none.png) 0;
 }

 QTreeView::branch:has-children:!has-siblings:closed,
 QTreeView::branch:closed:has-children:has-siblings {
         border-image: none;
         image: url(:/tulip/gui/ui/right_arrow-black.png);
 }

 QTreeView::branch:open:has-children:!has-siblings,
 QTreeView::branch:open:has-children:has-siblings  {
         border-image: none;
         image: url(:/tulip/gui/ui/down_arrow-black.png);
 }</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="rightFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>80</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="handleWidth">
             <number>1</number>
            </property>
            <property name="childrenCollapsible">
             <bool>false</bool>
            </property>
            <widget class="QFrame" name="parametersFrame">
             <layout class="QVBoxLayout" name="rLayout2">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="parametersLabel">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;b&gt;Parameters&lt;b&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="parameters">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::AllEditTriggers</set>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <attribute name="horizontalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>200</number>
                </attribute>
                <attribute name="horizontalHeaderMinimumSectionSize">
                 <number>150</number>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>true</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tulip/ClearableLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ExportWizard</sender>
   <signal>currentIdChanged(int)</signal>
   <receiver>ExportWizard</receiver>
   <slot>updateFinishButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>ExportWizard</receiver>
   <slot>browseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>847</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ExportWizard</receiver>
   <slot>pathChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>groupSelected(QString)</slot>
  <slot>algorithmSelected(QString)</slot>
  <slot>updateFinishButton()</slot>
  <slot>pathChanged(QString)</slot>
  <slot>browseButtonClicked()</slot>
 </slots>
</ui>
